beeswax/openrtb/openrtb.proto

```diff
@@ -761,6 +761,18 @@ message BidRequest {
      * Language of the content. Use alpha-2/ISO 639-1 codes.
      */
     optional string language = 19;
+    /*
+     * Genre that best describes the content (e.g., rock, pop, etc).
+     */
+    optional string genre = 20;
+    /*
+     * Network Object
+     */
+    optional Network network = 21;
+    /*
+     * Channel Object
+     */
+    optional Channel channel = 22;
```

Comment:
Are these related?

Reply:
Just pulling in the latest version of the pb that we're using internally -- should we not expose these elements?

Reply:
That's fine.